src/rail_map.h

```diff
@@ -530,7 +530,25 @@ static inline void MakeRailNormal(Tile t, Owner o, TrackBits b, RailType r)
 	t.m8() = r;
 }
 
+/**
+ * Sets the exit direction of a rail depot.
+ * @param t     Tile of the depot.
+ * @param dir   Direction of the depot exit.
+ */
+static inline void SetRailDepotExitDirection(Tile t, DiagDirection d)
+{
+	assert(IsRailDepotTile(t));
+	t.m5() = RAIL_TILE_DEPOT << 6 | d;
+}
 
+/**
+ * Make a rail depot.
+ * @param t     Tile to make a depot on.
+ * @param owner New owner of the depot.
+ * @param did   New depot ID.
+ * @param dir   Direction of the depot exit.
+ * @param rt    Road type of the depot.
```

Comment:
`owner`, `dir`, and `rt` don't match implementation. As in my comment above, the longer versions are nicer. 🙂

Reply:
Changed all parameter names to their longer versions :)